tests/libc_newlib/README.md

```diff
@@ -0,0 +1,19 @@
+About
+=====
+
+Terifies if newlib/newlib-nano is correctly included by the build system
```

Comment:
Can you include instructions on how to force a failure of this test? Then people can test the test.

Reply:
The link time test (currently done at runtime) can be tested by commenting this line:

https://github.com/RIOT-OS/RIOT/blob/db6cb1ac65e1fde042c511e68ef13c42bf4b477e/makefiles/libc/newlib.mk#L17

And the compile time test can be tested by commenting this one: https://github.com/RIOT-OS/RIOT/blob/db6cb1ac65e1fde042c511e68ef13c42bf4b477e/makefiles/libc/newlib.mk#L80

Reply:
Tested both tests and it failed as expected.